apps/testingaccessibility/src/pages/api/callback.ts

```diff
@@ -0,0 +1,38 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import {withSentry} from '@sentry/nextjs'
+
+const callback = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method === 'GET') {
+    try {
+      const response = `
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing Accessibility Login Redirect</title>
+    <script>
+      const url = new URL(window.location.href)
+      url.pathname = "/api/auth/callback/email"
+      window.location.href = url.href
```

Comment:
rewrite the URL back to the default next-auth email callback route and send them on their way

this assumes most of these bots aren't executing JS, which is a big assumption, but is likely to catch at least a good number of systems that attempt this sort of thing

usually these systems are protecting against "detonation" attacks where clicking a link causes immediate harm. This "trick" here seems like a pretty low-rent hack against that sort of high-end protection 🤡

might not even be worth it?